solr/CHANGES.txt

```diff
@@ -107,6 +107,8 @@ Improvements
 
 * SOLR-16927: Allow SolrClientCache clients to use Jetty HTTP2 clients (Alex Deparvu, David Smiley)
 
+* SOLR-16897: Add support of OAuth 2.0/OIDC 'code with PKCE' flow - back-end  (Lamine Idjeraoui. Review and contribution by janhoy. Review by Kevin Risden)
```

Comment:
Simply list the contributors, no need to qualify what each one contributed.

Github will set author and add `Co-authored-by: John Doe <john@doe.com>` for code contributors in final merge message.

Also, no ned for two CHANGES entris even if you split it in two PRs with two commits. Suggest to add both JIRA numbers in same line:

```suggestion
* SOLR-16896, SOLR-16897: Add support of OAuth 2.0/OIDC 'code with PKCE' flow (Lamine Idjeraoui, janhoy, Kevin Risden)
```